src/main/java/bio/terra/service/load/LoadService.java

```diff
@@ -0,0 +1,55 @@
+package bio.terra.service.load;
+
+import bio.terra.service.load.exception.LoadLockFailureException;
+import bio.terra.service.load.flight.LoadMapKeys;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+
+@Component
+public class LoadService {
+    private final LoadDao loadDao;
+
+    @Autowired
+    public LoadService(LoadDao loadDao) {
+        this.loadDao = loadDao;
+    }
+
+    public void lockLoad(String loadTag, String flightId) {
+        loadDao.lockLoad(loadTag, flightId);
+    }
+
+    public void unlockLoad(String loadTag, String flightId) {
+        loadDao.unlockLoad(loadTag, flightId);
+    }
+
+    /**
+     * @param inputTag  may be null or blank
+     * @return either valid inputTag or generated date-time tag.
+     */
+    public String computeLoadTag(String inputTag) {
```

Comment:
Could this method be static?

Reply:
Yes, but since LoadService is a @Component, all the methods have just the single instance. Do you see a benefit in making that one in particular static?

Reply:
No benefit, just a question of convention. At previous projects, I've worked under the convention that if a method doesn't use any instance-specific properties, then it is declared static. None of these projects used Spring though, and I understand our group may have its own conventions, so I'm okay either way.

Reply:
That is a sensible convention. I'm not sure it applies in Spring-land.

The only argument I can come up for using `static` inside of an @component is if there were places that used just the static method and nothing else from the component. Then it would save that case of Spring dynamic linking.

@thathert Do you have an opinion? I don't feel strongly about it, but we should be consisitent.